UC4-chaincode/src/main/java/de/upb/cs/uc4/chaincode/contract/group/GroupContract.java

```diff
@@ -56,6 +56,7 @@ public String addUserToGroup(final Context ctx, String enrollmentId, String grou
 
         try {
             cUtil.validateApprovals(ctx, contractName, transactionName, new String[]{enrollmentId, groupId});
+            cUtil.validateAttributes(ctx, new ArrayList<String>() {{add("sysAdmin");}});
```

Comment:
"sysAdmin" should be stored in some static final variable, like groups (like group names in AccessManager)

Reply:
resolved in new commits